src/modules/job-manager/plugins/perilog.c

```diff
@@ -363,9 +359,14 @@ static int run_command (flux_plugin_t *p,
         return -1;
     }
 
+    if (!(proc = perilog_proc_create (p, id, prolog))) {
+        flux_log_error (h, "%s: proc_create", prolog ? "prolog" : "epilog");
+        goto error;
+    }
+
     if (flux_cmd_setenvf (cmd, 1, "FLUX_JOB_ID", "%s", idf58 (id)) < 0
         || flux_cmd_setenvf (cmd, 1, "FLUX_JOB_USERID", "%u", userid) < 0) {
-        flux_log_error (h, "%s: flux_cmd_create", prolog ? "prolog" : "epilog");
+        flux_log_error (h, "%s: flux_cmd_create", perilog_proc_name (proc));
```

Comment:
Minor: mention the switch to `perilog_proc_name()` in the commit description or perhaps split the commit